gemfiles/rails_5_2.gemfile

```diff
@@ -3,12 +3,6 @@
 source "http://rubygems.org"
 
 gem "rails", "~> 5.2.0"
-gem "rubocop"
-
-group :local do
-  gem "guard-rspec", "~> 4.2"
-  gem "pry"
-end
```

Comment:
I am not sure why these gems where here in the first place. They are not present in the `Appraisals` file so I wasn't sure if I should re-add them manually.